### PR TITLE
Doc: add prefetch to plugin index page

### PR DESCRIPTION
**Pull request by mlibbey**
The Prefetch plugin is mentioned in the Table of Content section, but not the body.

### CODE REVIEW

File: doc/admin-guide/plugins/index.en.rst [hunk 1]
```diff
@@ -220,6 +220,9 @@ directory of the |TS| source tree. Experimental plugins can be compiled by passi
 :doc:`MySQL Remap <mysql_remap.en>`
    Allows dynamic remaps from a MySQL database.
 
+:doc:`Prefetch <prefetch.en.en>`
```
Comment (maskit) [inline doc/admin-guide/plugins/index.en.rst]: Two en?

Comment (mlibbey) [reply doc/admin-guide/plugins/index.en.rst]: Thanks!

**Comment by randall** [comment]
[approve ci clang-analyzer]

**Comment by bryancall** [comment]
Cherry-picked into the 8.0.6 release.

**Pull request merged**